README.md

```diff
@@ -3,17 +3,17 @@
 This repository contains a suite of tools that can be used to manage key areas
 of your GCP organization.
 
-- Deploy
-- Monitor
-- Audit
+-   Deploy
```

Comment:
vs code markdown lint reported this as a problem, can you double check with it?

Reply:
Hmm. Which markdown formatter are you using?

Reply:
https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint

Reply:
(not sure if this is the best one to use, it was just recommended by VS code)

Reply:
https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md030

Reply:
I used https://github.com/igorshubovych/markdownlint-cli to format it accordingly.